### PR TITLE
Corrections to controller docs before 2.5 GA

### PR DESCRIPTION
**Pull request by EMcWhinn**
Pre-release fixes for Controller docs

https://issues.redhat.com/browse/AAP-23339

Affects:

`titles/controller-user-guide`
`titles/controller-admin-guide`
`titles/controller-getting-started-guide`

### CODE REVIEW

File: downstream/modules/platform/proc-controller-importing-subscriptions.adoc [hunk 1]
```diff
@@ -6,12 +6,12 @@ After you have obtained an authorized {PlatformNameShort} subscription, you must
 .Prerequisites
 
 * You have obtained a subscriptions manifest. 
-For more information, see xref:proc-controller-obtaining-subscriptions-manifest[Obtaining a subscriptions manifest].
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/getting_started_with_automation_controller/index#controller-managing-subscriptions[Obtaining a subscriptions manifest].
```
Comment (michellemacrh) [inline downstream/modules/platform/proc-controller-importing-subscriptions.adoc:9]: May be worth considering using the {BaseURL} attribute here and {PlatformVers} to future proof the links

**Review by michellemacrh — COMMENTED** [review]
1 small comment, otherwise LGTM

File: downstream/modules/platform/proc-controller-importing-subscriptions.adoc (hunk 1 above)
Comment (EMcWhinn) [reply downstream/modules/platform/proc-controller-importing-subscriptions.adoc]: Good spot, updated.

**Comment by michellemacrh** [comment]
LGTM

**Pull request merged**